src/pages/press.tsx

```diff
@@ -0,0 +1,235 @@
+import styled from '@emotion/styled';
+import { getEmSize } from '../styles/mixins';
+import { colors, breakpoints } from '../styles/variables';
+import mediaKitPreview from '../resources/media-kit.png';
+import mediaKitPDF from '../resources/Media_kit2.pdf';
+import LogoWhite from '../resources/logo-white.svg';
+import LogoBlack from '../resources/logo-black.svg';
+import Logo from '../resources/logo-textless.svg';
+
+const MediaKit = styled.div`
+    display: flex;
+    flex-wrap: wrap;
+    width: 100%;
+    margin-bottom: 100px;
+
+    div {
+        @media(min-width: ${getEmSize(breakpoints.md)}em) {
+            flex: 5;
+        }
+        @media(max-width: ${getEmSize(breakpoints.md) - 1}em) {
+            margin: auto;
+        }
+    }
+
+    .preview-img {
+        width: 300px;
+        @media(min-width: ${getEmSize(breakpoints.md)}em) {
+            transform: translateY(15px);
+        }
+    }
+
+    p {
+        @media(max-width: ${getEmSize(breakpoints.md) - 1}em) {
+            text-align: center;
+        }
+    }
+
+    .logo {
+        display: table;
+        border-collapse:separate;
+        border-spacing:0 15px;
+        width: 100%;
+    }
+
+    .svg-container {
+        display: table-cell;
+        vertical-align: middle;
+        height: 120px;
+        width: 25%;
+        background: #2a2a2a;
+        text-align: center;
+        @media(max-width: ${getEmSize(breakpoints.md) - 1}em) {
+            width: 35%;
+        }
+        &:not(:last-child) {
+            margin-bottom: 30px;
+        }
+
+        & a {
+            display: inline-block;
+            width: 100%;
+            height: 100%;
+        }
+    }
+
+    span {
+        display: table-cell;
+        vertical-align: middle;
+        width: 72%;
+        padding-left: 60px;
+        @media(max-width: ${getEmSize(breakpoints.md) - 1}em) {
+            width: 75%;
+        }
+    }
+
+    svg {
+        height: 30px;
+        transform: translateY(45px);
+    }s
+`
+
+const PressPage: React.SFC<{}> = () => (
+    <IndexLayout title="Gitpod - Press" canonical="https://www.gitpod.io/press">
+        <Page>
+            <Container>
+                <Logos logos={[
+                    [1050, 125, 40],
+                ]} />
+                <div style={{ marginTop: 60 }}>
+                    <h4 style={{ color: colors.fontColor2 }}>Media kit</h4>
+                    <h1>Press Information</h1>
+                    <UnderLine />
+                </div>
+                <p style={{ marginBottom: 80 }} >
+                    Would you like to cooperate with us? Write something about Gitpod? Or maybe you are just curious?<br />Our media kit will give an overview about Gitpod.
+                </p>
+                <MediaKit>
+                    <div style={{ position: 'relative' }}>
+                        <a href={mediaKitPDF} download>
+                            <img src={mediaKitPreview} alt="Media Kit PDF Preview" className="preview-img" />
```

Comment:
I think that technically, the "media kit" is this entire page (i.e. the "fact sheet PDF" + the logos).

```suggestion
                            <img src={mediaKitPreview} alt="Fact Sheet PDF Preview" className="preview-img" />
```